var/spack/repos/builtin/packages/mesa/package.py

```diff
@@ -64,10 +68,15 @@ class Mesa(MesonPackage):
     variant('opengles', default=False, description="Enable OpenGL ES support.")
 
     # Provides
-    provides('gl@4.5',  when='+opengl')
-    provides('glx@1.4', when='+glx')
-    # provides('egl@1.5', when='+egl')
-    provides('osmesa', when='+osmesa')
+    provides('gl@4.5',  when='+opengl ~glvnd')
+    provides('glx@1.4', when='+glx ~glvnd')
+    # This isn't possible b/c of conflict. Should remove?
+    # provides('egl@1.5', when='+egl ~glvnd')
```

Comment:
`egl` can still be provided when `+egl ~glvnd`.  The conflict for `+egl +opengl ~glvnd` will take care of the invalid configuration.